### DISCUSSION TITLE
Blog lists/bundles

### DISCUSSION THREAD

**Discussion by martienne17** [AprilSylph/XKit-Rewritten#1252] [Suggestion Box]
An idea taken from twitter's list function, but I'm wondering if this kind of thing would be possible in x-kit. I'm mainly wondering because I have ended up following over a hundred tournament blogs and I comb my following list about once a week to make sure I've voted in all the polls. But I also have a lot of topics I follow different blogs for (ex-religious content, star trek, writing, etc.) and being able to check by list would be really helpful.

**Comment by marcustyphoon** [comment]
Tumblr will very possibly implement this themselves!

You can actually try out the backend code already. Try this link: https://www.tumblr.com/timeline/blogpack?blogs=staff,changes

**Reply by martienne17** [reply]
Oh, thanks for showing me! It's a little fussy since you have to add all the urls yourself but I imagine the final version of it will have something in the gui?

**Comment by martienne17** [comment]
I turned off labs and set up a queue of 12 test posts to see if any
disappear. I just went to the mass post editor and there are probably 100
posts still stuck in there that I don't see when I open the queue page. How
do I get them back?
